src/hazelcore/util/OptUtil.re

```diff
@@ -1,33 +1,16 @@
-// TODO prune redundant functions now that
-// we're on 4.08 and can use stdlib Option
-
-let map = (f: 'a => 'b, opt: option('a)): option('b) =>
-  switch (opt) {
-  | None => None
-  | Some(a) => Some(f(a))
-  };
 let map2 =
     (f: ('a, 'b) => 'c, opt1: option('a), opt2: option('b)): option('c) =>
   switch (opt1, opt2) {
   | (None, _)
   | (_, None) => None
   | (Some(a), Some(b)) => Some(f(a, b))
   };
-let map_default = (~default: 'b, f: 'a => 'b, opt: option('a)): 'b =>
-  switch (opt) {
-  | None => default
-  | Some(a) => f(a)
-  };
+
 let get = (if_absent: unit => 'a, opt: option('a)): 'a =>
```

Comment:
this has a name clash with a different function `Option.get`. it behaves most closely to `Option.value`, except the default is thunked. maybe call it `OptUtil.fvalue` or `OptUtil.thvalue` (whatever you like)

Reply:
I say we call it `get` since we should always use this instead of `Option.get`